YugaByte/yugabyte-db
# ENG-5289 #2025 [YSQL][docs] DO statement

Add description for `DO` command into `latest/api/ysql/commands/` YB docs section.
Description and example are taken from Postgres's official doc 

Test plan:
Manual
- start docs web browser
- check `do` command in command list at `latest/api/ysql/commands/` URL
- check link for `do` is clickable and leads to working page with detailed description
